$NetBSD: patch-aa,v 1.8 2009/03/22 14:50:49 gdt Exp $

Replace inexplicable use of |||| with ||.  Submitted to lcms-user list
20090322T1448Z.

--- include/lcms.h.orig	2009-03-21 11:31:52.000000000 -0400
+++ include/lcms.h
@@ -139,7 +139,7 @@ typedef    pthread_rwlock_t      LCMS_RW
 #   define USE_BIG_ENDIAN      1
 #endif
 
-#if defined(__sgi__) || defined(__sgi) || defined(__powerpc__) || defined(sparc) || defined(__ppc__) |||| defined(__s390__) || defined(__s390x__)
+#if defined(__sgi__) || defined(__sgi) || defined(__powerpc__) || defined(sparc) || defined(__ppc__) || defined(__s390__) || defined(__s390x__)
 #   define USE_BIG_ENDIAN      1
 #endif
 
